routes/program.js

```diff
@@ -36,6 +37,18 @@ module.exports = {
 
 		response.redirect('/program');
 	},
+	viewProgramPage: async function (request, response) {
+		let programId = request.params.id;
+		let programObj = await db.getProgramById(programId);
+		let renderData = {
+			program: programObj,
+			add: false,
+			view: true,
+			students: programObj.student_list,
```

Comment:
After updating the `student_list` schema field, you'll have to get the objects from the object ids here